### PR TITLE
feat: [SRM-11276]: Default Change Source in for MS Template

### PR DESCRIPTION
**Pull request by deepesh-ui**
### Summary 
 Load default change source in MS template 

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by deepesh-ui** [comment]
retrigger titlecheck

File: src/modules/85-cv/pages/monitored-service/components/Configurations/Configurations.tsx [hunk 1]
```diff
@@ -153,26 +153,34 @@ export default function Configurations(
...
+        })
+      }
     }
   }, [yamlMonitoredService])
 
   useEffect(() => {
     if (identifier) {
       fetchMonitoredService()
-    } else {
+    } else if (isTemplate ? templateIdentifier === '-1' : !identifier) {
```
Comment (praneshg239) [inline src/modules/85-cv/pages/monitored-service/components/Configurations/Configurations.tsx:183]:
```ts
(isTemplate && templateIdentifier === '-1') || (!isTemplate && !identifier)
```

File: src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.utils.ts [hunk 2]
```diff
@@ -13,15 +13,13 @@ import { MonitoredServiceType } from './components/MonitoredServiceOverview/Moni
 import type { MonitoredServiceForm } from './Service.types'
 
 export const getInitFormData = (
-  data: MonitoredServiceDTO | undefined,
+  data: MonitoredServiceDTO | NGMonitoredServiceTemplateInfoConfig | undefined,
```
Comment (praneshg239) [inline src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.utils.ts:16]: `data?:`

**Review by praneshg239 — DISMISSED** [review]

File: src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.utils.ts (hunk 2 above)
Comment (deepesh-ui) [reply src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.utils.ts]: fixed , made data Optional property

File: src/modules/85-cv/pages/monitored-service/components/Configurations/Configurations.tsx (hunk 1 above)
Comment (deepesh-ui) [reply src/modules/85-cv/pages/monitored-service/components/Configurations/Configurations.tsx]: Updated logic to as suggested snippet

**Review by vasuki-kalluri — APPROVED** [review]

**Review by praneshg239 — APPROVED** [review]

**Pull request merged**